lib/helper.js

```diff
@@ -1,113 +1,100 @@
 'use strict';
-var clone = require('clone');
```

Comment:
Better approach will be used https://github.com/seznam/IMA.js-helpers as dependency.